firmware/robot/cpu/encoder_monitor.c

```diff
@@ -46,32 +46,35 @@ void encoder_monitor()
 		// 	}
 		// }
 	} else {
+        // THIS IS A HACK
```

Comment:
Yeah I don't think there's much chance of us ever adding encoders back to the 2011 bots, so it should be fine to keep.